Formula/enzyme.rb

```diff
@@ -17,7 +16,7 @@ class Enzyme < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "llvm@14"
+  depends_on "llvm"
```

Comment:
They don't have CI for anything newer than LLVM 14, so it's unlikely that this works with the latest LLVM. https://github.com/EnzymeAD/Enzyme/blob/5cfe075070d63cf09768ca3a8be811e28760d2af/.github/workflows/enzyme-ci.yml#L13

Reply:
However I've found commits mentioning compatibility with LLVM 15: https://github.com/EnzymeAD/Enzyme/search?q=llvm+15&type=commits and a PR telling it fixed build with LLVM 15: https://github.com/EnzymeAD/Enzyme/pull/818